scripts/xtp_modify_jobfile.in

```diff
@@ -0,0 +1,191 @@
+#! /usr/bin/env python3
+#
+# Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+VERSION = '@PROJECT_VERSION@ #CSG_GIT_ID#'
+import sys
+import argparse
+import re
+import lxml.etree as lxml
+import fnmatch
+
+
+PROGTITLE = 'THE VOTCA::XTP MODIFY JOBFILE'
+PROGDESCR = 'Creates a subset Jobfile from a larger Jobfile using selection criteria'
+VOTCAHEADER = '''\
```

Comment:
Python now support [f-strings](https://realpython.com/python-f-strings/), so the string:
```python
s = "some {version}".format(version=Version)
```
Can be conveniently written as:
```python
s = f"some {VERSION}"
```

Reply:
since when does it? I like the more explicit version

Reply:
Since Python 3.6 and `format` is now considered *old school* ;)

Reply:
hmmm which is the oldest python version we support?

Reply:
I don't know. I use f-string on the other scripts that I have done for the documentation